server/sonar-web/src/main/js/apps/overview/events/PreviewGraph.js

```diff
@@ -97,23 +101,23 @@ export default class PreviewGraph extends React.PureComponent {
     }
   }
 
-  formatValue = (tick: number | string) =>
+  formatValue = (tick /*: number | string */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)